dev-python/bokeh/files/bokeh-0.7.1-setup.patch

```diff
@@ -0,0 +1,10 @@
+--- setup.py	2015-01-12 19:37:38.000000000 +0100
```

Comment:
Oh, I wasn't aware of that and actually nearly no of the patches if looked into so far come with a header. I will do this the other patches (other ebuilds) I pushed so far too.